src/jit/instrsxarch.h

```diff
@@ -196,6 +196,7 @@ INST3( cvttsd2si,   "cvttsd2si"   , 0, IUM_WR, 0, 0, BAD_CODE,     BAD_CODE, SSE
 
 #ifndef LEGACY_BACKEND
 INST3( movntdq,     "movntdq"     , 0, IUM_WR, 0, 0, PCKDBL(0xE7), BAD_CODE, BAD_CODE)
+INST3( movnti,      "movnti"      , 0, IUM_WR, 0, 0, PCKFLT(0xC3), BAD_CODE, BAD_CODE)  // Move doubleword/quadword from r32/r64 to m32/m64 using nontemporal hint
```

Comment:
This doesn't appear to be used anywhere

Reply:
It slipped in from my work on `StoreNonTemporal` implementation, I can remove and reintroduce in the next PR. What is your preference?

Reply:
Introducing in the PR where it is consumed is my preference.